src/graphics/texture.js

```diff
@@ -117,6 +123,9 @@ class Texture {
         this._format = PIXELFORMAT_R8_G8_B8_A8;
         this.type = TEXTURETYPE_DEFAULT;
 
+        // texture projection type, used only when texture needs to be projected
+        this.projection = TEXTUREPROJECTION_EQUIRECT;
```

Comment:
I don't really like the fact that 99% of textures in a project are going to be regular 2D textures with no projection. If I query the projection of such a texture, I would expect 'none' instead of 'equirect'.

Reply:
I agree but @slimbuck changed my mind. I'll let him state the argument, as I'd prefer to have the NONE option too.

Reply:
Ooo. 2 against 1. Bring it on. 👊

Reply:
It seems a pain to handle the option of NONE everywhere projection is used, but I agree users might wonder why their picture of a doggie has been assigned EQUIRECT projection :)

So let's add NONE and then default to EQUIRECT :+1:

Reply:
Shouldn't NONE be the default? And if you start screwing around with projections, then you have to set one as desired?

Reply:
Yep, sounds good to me. Don't you just love a happy consensus? 😄

Reply:
done, added TEXTUREPROJECTION_NONE